kinder/ci/workflows/skew-x-on-y-tasks.yaml

```diff
@@ -7,35 +7,39 @@ vars:
   # vars defines default values for variable used by tasks in this workflow;
   # those values might be overridden when importing this files.
   kubeadmVersion: v1.13.5
+  kubeletVersion: v1.12.8
   kubernetesVersion: v1.12.8
   controlPlaneNodes: 1
   workerNodes: 2
   baseImage: kindest/base:v20190403-1ebf15f
   image: kindest/node:test
   clusterName: kinder-xony
   kubeadmVerbosity: 6
+  defaultIgnorePreflightErrors: Swap,SystemVerification,FileContent--proc-sys-net-bridge-bridge-nf-call-iptables
+  ignorePreflightErrors: ""
 tasks:
-- name: pull-base-image
-  description: |
-    pulls kindest/base image with docker in docker and all the prerequisites necessary for running kind(er)
-  cmd: docker
-  args:
-    - pull
-    - "{{ .vars.baseImage }}"
-- name: add-kubernetes-versions
-  description: |
-    creates a node-image-variant by adding a Kubernetes version and
-    and a different Kubernetes version
-  cmd: kinder
-  args:
-    - build
-    - node-image-variant
-    - --base-image={{ .vars.baseImage }}
-    - --image={{ .vars.image }}
-    - --with-init-artifacts={{ .vars.kubernetesVersion }}
-    - --with-kubeadm={{ .vars.kubeadmVersion }}
-    - --loglevel=debug
-  timeout: 15m
+# - name: pull-base-image
```

Comment:
left-over from local testing?

Reply:
indeed, i blame the late hours working on this.